R/drive_cp.R

```diff
@@ -11,6 +11,7 @@
 #' @template name
 #' @templateVar name file
 #' @templateVar default Defaults to "Copy of `FILE-NAME`".
+#' @param ... Parameters to pass along to the API query.
```

Comment:
I think there should also be at least one example that uses `...`. For our own future selves, if nothing else.